pages/services.js

```diff
@@ -93,6 +97,95 @@ const Services = () => (
           },
         ]}
       />
+      <Layout nested={true} type="2-1" border={false}>
+        <div>
+          <a name="content-block-Mmedia" data-title="Content Block Media" />
```

Comment:
Could you please remove this capital letter `M` in `content-block-Mmedia`?

Reply:
Done @caiobozato.

Reply:
Done.